frontend/src/Editor/Components/components.js

```diff
@@ -151,6 +152,7 @@ export const componentTypes = [
       styles: {
         backgroundColor: { value: '#375FCF' },
         textColor: { value: '#fff' },
+        loaderColor: { value: '#fff' },
```

Comment:
@sherfin94 is it okay to set the default color as `#fff` ?

Reply:
Yes, the default loader color is `#fff`.

@abhinabaadhikari has also nullish coalesced the value to `#fff` in code, so backward compatibility is also handled